windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -29,7 +29,9 @@ class RegisteredCallbackToken
 
 UwbConnector::UwbConnector(std::string deviceName) :
     m_deviceName(std::move(deviceName))
-{}
+{
+    NotificationListenerStart();

```

Comment:
Why is this needed here? It seemed like the callback registration code would take care of calling this when necessary.

Reply:
This function is actually not called by anything else, as its only role is to start the wait process so I think the constructor will have to call it. Another way, is actually to get rid of the condition variable and just have the callback registration codes grab the mutex, check if the were callbacks registered before, and if there were none then also start the notification listener. I think this method actually makes a lot of sense

Reply:
#203